src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependencyNode.cs

```diff
@@ -114,6 +114,8 @@ internal DependencyNode()
                         : UnresolvedDependencyFlags.Union(flags);
         }
 
+        private object _childrenLock = new object();
```

Comment:
yes, it should be readonly

---
In reply to: [99721132](https://github.com/dotnet/roslyn-project-system/pull/1469#discussion_r99721132) [](ancestors = 99721132)